x/superfluid/keeper/epoch.go

```diff
@@ -63,10 +61,6 @@ func (k Keeper) MoveSuperfluidDelegationRewardToGauges(ctx sdk.Context) {
 		// we use cacheCtx and apply the changes later
 		_ = osmoutils.ApplyFuncIfNoError(ctx, func(cacheCtx sdk.Context) error {
 			_, err := k.dk.WithdrawDelegationRewards(cacheCtx, addr, valAddr)
-			if errors.Is(err, distributiontypes.ErrEmptyDelegationDistInfo) {
```

Comment:
just for context, why was this removed?

Reply:
This line wasn't here in the first place, this was recently added to better log errors. However, with the main problem that caused all these errors fixed(by deleting intermediary account), we could revert it back to the code where we didn't have this line. 

Could still leave logging errors if necessary